templates/servicesTab.hbs

```diff
@@ -0,0 +1,8 @@
+
+  <div class="tab__name">

```

Comment:
@JFozzie Can you update this using a button tag? This will improve accessibility.